products/compute/api.yaml

```diff
@@ -1801,7 +1803,7 @@ objects:
       - !ruby/object:Api::Type::NestedObject
         name: 'consistentHash'
         description: |
-          Consistent Hash-based load balancing can be used to provide soft session
+          Consistent hash-based load balancing can be used to provide soft session
```

Comment:
This one comes from https://cloud.google.com/compute/docs/reference/rest/v1/backendServices, so I think we want to preserve it.
```suggestion
          Consistent Hash-based load balancing can be used to provide soft session
```

Reply:
Done. :raised_eyebrow: